src/utils/__tests__/safeRename.spec.js

```diff
@@ -0,0 +1,79 @@
+import childProcess from 'child_process';
+import fsPromises from 'fs/promises';
+import fs from 'fs';
+import os from 'os';
+import { join, resolve } from 'path';
+import { remove as extraRemove } from 'fs-extra';
+
+import safeRename from '../safeRename.js';
+
+const assetsDir = resolve('./src/utils/__tests__/assets/safeRename');
+
+/* input tar file contents:
+ * rename1.txt
```

Comment:
The tar file appears to also have a `rename2.txt`.

Reply:
Has been fixed.